server/apolloServer.js

```diff
@@ -13,7 +14,13 @@ function newApolloServer (app, schema, httpServer, tracing, playgroundConfig, gr
     schema,
     validationRules: [
       queryDepthLimit(serverSecurity.queryDepthLimit),
-      createComplexityLimitRule(serverSecurity.complexityLimit)
+      createComplexityLimitRule(serverSecurity.complexityLimit, {
+        formatErrorMessage: (cost) => {
+          const queryLog = 'query with cost ' + cost + ' exceeds complexity limit'
```

Comment:
two really minor nits

* `queryLog` is not the most descriptive name. Perhaps something like `complexityErrorMessage` would be more descriptive.
* es6 template strings are more desirable over string concatenation.

```
const complexityErrorMessage = `query with ${cost} exceeds complexity limit`
```

Reply:
I'll change the name. With the templating I actually had what you have suggested originally but our linter doesn't like that...

```
[eslint] Unexpected template string expression. (no-template-curly-in-string)
```

Reply:
You need to use backticks for template strings. e.g \`some string\` 

That linter error is basically saying 'hey it looks like you're trying to do a template string but you're using normal quotes'

Reply:
🤦‍♂️